src/components/CircleDetails.vue

```diff
@@ -388,6 +388,18 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+.app-content-details {
+	padding: calc(var(--default-grid-baseline) * 2);
+	padding-top: 0;
+	height: 100%;
+	overflow: auto;
+
+	// Compensate top padding reserved for the back button on mobile
+	@media (max-width: 1024px) {
+		height: calc(100% - 44px);
+		padding: 0 20px;
+	}
+}
```

Comment:
This styling also needs to be applied to other components (not just `CircleDetails`), e.g. `ContactDetails`.